distribution/rpm/src/main/packaging/init.d/elasticsearch

```diff
@@ -35,8 +35,6 @@ fi
 ES_HOME="/usr/share/elasticsearch"
 MAX_OPEN_FILES=65536
 MAX_MAP_COUNT=262144
-LOG_DIR="/var/log/elasticsearch"
```

Comment:
Should we fail hard if these env vars are set? Not sure where the appropriate place to check it is. Somewhere in the service files?